src/components/buttonMenu/buttonMenu.stories.svelte

```diff
@@ -12,7 +12,15 @@
   const handleAction = () => console.log('action')
 </script>
 
-<Meta title="Components/ButtonMenu" component={ButtonMenu} />
+<Meta
+  title="Components/ButtonMenu"
+  component={ButtonMenu}
+  argTypes={{
+    '--leo-menu-control-width': {
```

Comment:
I don't think this exists anymore

Reply:
This one still exists. It's a read only prop to know the width of the control. https://github.com/brave/leo/blob/ecc35e8b9f5f8cce355d7b8da0bec86ee3cf7ebf/src/components/menu/menu.svelte#L184